#!/usr/bin/env python
PACKAGE = "origin_transform"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("offset_x", double_t, 0, "Offset in x direction", 0.0, -100.0, 100.0)
gen.add("offset_y", double_t, 0, "Offset in y direction", 0.0, -100.0, 100.0)
gen.add("offset_yaw", double_t, 0, "Yaw rotation offset in radian", 0.0, -3.14159, 3.14159)

exit(gen.generate(PACKAGE, "origin_transform", "OriginTransform"))